docs/docs-content/clusters/pcg/deploy-app-pcg.md

```diff
@@ -0,0 +1,641 @@
+| Description | Cluster profile as part of the Deploy App Workloads with a PCG tutorial.                                                                                    |
+| Type        | Full                                                                                                                                                        |
+| Tags        | `spectro-cloud-education`, `app:hello-universe`, `terraform_managed:false`, `repository:spectrocloud:tutorials`, `tutorial:DEPLOY_APP_WORKLOADS_WITH_A_PCG` |
+
+Click on **Next** to continue.
+
+**Cloud Type**
+
+In the **Cloud Type** section, select VMware as the infrastructure provider, then click **Next** to proceed.
+
+**Profile Layers**
+
+Add the following core infrastructure layers displayed to your profile.
+
+| Pack Name          | Version | Layer            |
+| ------------------ | ------- | ---------------- |
+| ubuntu-vsphere LTS | 22.4.x  | Operating System |
+| kubernetes         | 1.28.x  | Kubernetes       |
+| cni-calico         | 3.26.x  | Network          |
+| csi-vsphere-csi    | 3.0.x   | Storage          |
+
+After adding each layer, click on the **Next Layer** button. Once the storage layer is added, click **Confirm** to
+complete the core infrastructure stack.
+
+Next, click on the **Add New Pack** button to include add-on layers to your cluster profile.
+
+Add the **MetalLB (Helm)** pack to your profile. The [MetalLB](../../integrations/metallb.md) pack provides a
+load-balancer implementation for your Kubernetes cluster, as VMware does not offer a load balancer solution natively.
+The load balancer is required to help the _LoadBalancer_ service specified in the Hello Universe application manifest
+obtain an IP address, so that you can access the application on your browser.
+
+| Pack Name       | Version | Layer         |
+| --------------- | ------- | ------------- |
+| lb-metallb-helm | 0.13.x  | Load Balancer |
+
+Now, under **Pack Details**, click on **Values** and replace the predefined `192.168.10.0/24` IP CIDR listed below the
+**addresses** line with a valid IP address or IP range from your VMware environment to be assigned to your load
+balancer. Next, click **Confirm & Create** to add the MetalLB pack.
+
+![Metallb Helm-based pack.](/clusters_pcg_deploy-app-pcg_metallb-pack.png)
+
+Finally, click again on the **Add New Pack** button to add the Hello Universe pack.
+
+| Pack Name      | Version | Layer       |
+| -------------- | ------- | ----------- |
+| hello-universe | 1.1.x   | App Service |
+
+Click on the **Confirm & Create** button and then click **Next**.
+
+**Review**
+
+Review the selected cluster layers and configurations. If everything looks correct, click on **Finish Configuration** to
+complete the cluster profile creation.
+
+![Cluster profile review page.](/clusters_pcg_deploy-app-pcg_cluster-profile.png)
+
+### Deploy a VMware Cluster
```

Comment:
I wonder if this entire deployment flow should be included in this tutorial or if we should defer to another deploy a cluster page https://docs.spectrocloud.com/clusters/data-center/vmware#deploy-a-vmware-cluster